app/pages/membership-stripe.vue

```diff
@@ -0,0 +1,147 @@
+<template>
+  <div>
+    <SubTitle title="Lahmacun membership" :maintitle="true" />
+    <div class="container my-8">
+      <div v-if="$fetchState.pending" class="center">
+        Loading...
+      </div>
+
+      <div v-if="membershipContent" class="max-w-4xl mx-auto">
+        <div class="mb-4">
+          <h2>{{ membershipContent.title.rendered }}</h2>
+        </div>
+        <div class="grid md:grid-cols-2 md:gap-8 gap-4">
+          <div>
+            <div v-sanitize="[sanitizeOptions, membershipContent.content.rendered]" />
+          </div>
+
+          <div>
+            <form action="https://cms.lahmacun.hu/wp-json/stripe/return_checkout_session_recurring_membership"
+              method="GET">
+              <div>
+                <label for="show_name">Select your show</label>
+                <div class="relative max-w-md bg-white rounded-sm">
+                  <select class="show-select" :class="{ showSelected: show_name.length > 1 }" @change="selectShow">
```

Comment:
@baxgas right now the show name is not sent along with the API call. I guess you need to add `name="show_name"` to the `select` element for that to happen.